frontend/src/store/modules/user.ts

```diff
@@ -28,6 +28,11 @@ export default {
   actions: {
     signOut() {
       firebase.auth().signOut();
+    },
+    async getUserClaims({ state }: any) {
```

Comment:
I believe this should go to the `getters` section.

Reply:
It was my understanding that `getters` were not used for async functions.